can_tools/scrapers/base.py

```diff
@@ -95,41 +90,61 @@ def quit_early(self, date: pd.Timestamp) -> bool:
 
 
 def _build_on_conflict_do_nothing_query(
-        df: pd.DataFrame, t_home: str, t_temp: str, pk: str
-    ):
+    df: pd.DataFrame, t_home: str, t_temp: str, pk: str, s_home: str = "data"
+) -> str:
     colnames = ", ".join(list(df))
     cols = "(" + colnames + ")"
     if not pk.startswith("("):
         pk = f"({pk})"
 
     return f"""
-    INSERT INTO data.{t_home} {cols}
+    INSERT INTO {s_home}.{t_home} {cols}
     SELECT {colnames} from {t_temp}
     ON CONFLICT {pk} DO NOTHING;
     """
 
 
+def _ensure_covid_demographics(connstr: str, df: pd.DataFrame):
```

Comment:
Did we decide to get rid of this method?

Reply:
Yes, will remove soon. Good catch